R/is_model.R

```diff
@@ -178,7 +178,7 @@ is_regression_model <- function(x) {
     "bamlss", "bamlss.frame", "brms",
     "cgam", "cgamm",
     "gam", "Gam", "GAMBoost", "gamlr", "gamlss", "gamm", "gamm4",
-    "stanmvreg", "stanreg"
+    "stanmvreg", "stanreg", "brmsfit"
```

Comment:
this should actually go into line 178, where we have a typo: `"brms"` should be `"brmsfit"`, I guess.